src/ripple/server/impl/ServerHandlerImp.cpp

```diff
@@ -521,7 +520,7 @@ struct ParsedPort
 
     boost::optional<boost::asio::ip::address> ip;
     boost::optional<std::uint16_t> port;
-    boost::optional<bool> allow_admin;
+    boost::optional<std::vector<beast::IP::Address>> admin_ip;
```

Comment:
@nbougalis  The syntax I wrote in the docs was modeled after the protocol section. In both cases brackets are optional, either way does not affect protocols or IPs specified.